### DISCUSSION TITLE
Yarn v3.0.1 workspaces are broken?

### DISCUSSION THREAD

**Discussion by mondo192** [yarnpkg/berry#3370] [Q&A] [answered]
Is there a bug with yarn 3 workspaces and typescript? I have setup a new project (might be useful for debugging) referenced from yarn 3 official docs [here](https://yarnpkg.com/getting-started/recipes). For a sanity check I can do this no problem with yarn v1 and lerna but not yarn berry. 

[REPO OF PROJECT](https://github.com/mondo192/reactexpress)

Steps 
```
mkdir reactexpress && cd reactexpress
yarn set version berry
yarn init -w 
yarn plugin import typescript
yarn packages/backend init
yarn packages/backend add express
yarn packages/backend add -D @types/node @types/express typescript
yarn packages/backend tsc --init
```

```
import http from "http"; <-- this fails due to type declarations missing
```
Project structure:
.
├── LICENSE
├── package.json
├── packages
│   ├── backend
│   │   ├── package.json
│   │   ├── README.md
│   │   ├── src
│   │   │   ├── app.ts
│   │   │   ├── routes
│   │   │   │   └── index.ts
│   │   │   └── server.ts
│   │   └── tsconfig.json
│   └── frontend
│       ├── package.json
│       └── README.md
├── README.md
└── yarn.lock

5 directories, 12 files

Then running ```yarn tsc --traceResolution``` from the root of the project results in: [gist](https://gist.github.com/mondo192/b73237fed86e166916528e3081bffc3b)

**Comment by merceyz** [comment, marked as answer]
Duplicate of https://github.com/yarnpkg/berry/issues/3350, see https://github.com/yarnpkg/berry/issues/3350#issuecomment-906793831 specifically
